src/presentation/http/router/noteSettings.test.ts

```diff
@@ -281,4 +281,63 @@ describe('NoteSettings API', () => {
 
     test.todo('Return 403 when user authorized, but not member of the team');
   });
+
+  describe('PATCH /note-teams/new-role/:newRole', () => {
+    test('Update team member role by user id and note id, with status code 200', async () => {
+      // patch member role of existing team member
+      const response = await global.api?.fakeRequest({
+        method: 'PATCH',
+        headers: {
+          authorization: `Bearer ${global.auth(1)}`,
+        },
+        url: '/note-settings/new-role/Pq1T9vc23Q/1/1',
```

Comment:
I guess it's better to pass new roles via body, instead of route params with should stay as resource (member) identifier

```
PATCH /note-settings/<note id>/member/<member id>

{
  role: 1
}
```

Reply:
or 

```
PATCH /note-settings/<note id>/team

{
  userId: 123,
  role: 1
}
```

Reply:
maybe team-role or smth, because we aren't really patching the team

Reply:
no, role is not a resource. It is a property of Team Member resource